scripts/ci/manage-host-operator.sh

```diff
@@ -85,7 +85,7 @@ read_arguments() {
 
 set -e
 
-read_arguments $@
+#read_arguments $@
```

Comment:
should this be removed?

Reply:
whoopsee, it's a leftover from my local testing :hand_over_mouth: 
good catch, thanks a3e7386

Reply:
https://issues.redhat.com/browse/KUBESAW-200